src/test/java/com/softserve/academy/spaced/repetition/service/UserCardQueueServiceTest.java

```diff
@@ -247,12 +261,14 @@ public void testUpdateExistingUserCardQueueUsingCardsPostponingWithNullStatusWhe
     @Test
     public void testUpdateExistingUserCardQueueUsingCardsPostponingWithNullStatusWhenStatusIsBadAndRememberingLevelIsMinimal()
             throws NotAuthorisedUserException {
-        UserCardQueueStatus status = UserCardQueueStatus.BAD;
-        RememberingLevel startingRememberingLevel = rememberingLevelRepository.findOne(1L);
-        RememberingLevel expectedRememberingLevel = rememberingLevelRepository.findOne(1L);
-        Date startingDateToRepeat = new Date();
         User mockedUser = createMockedUser(LearningRegime.CARDS_POSTPONING_USING_SPACED_REPETITION);
         when(mockedUserService.getAuthorizedUser()).thenReturn(mockedUser);
+        UserCardQueueStatus status = UserCardQueueStatus.BAD;
```

Comment:
This file became too long. Please consider using http://junit.org/junit4/javadoc/4.12/org/junit/runners/Parameterized.html

Reply:
This is quite a big task, so it'll take me some time to implement it. And I have a question for you about including SpringRunner instead of Parameterized runner, which I'll ask you in Thursday.

Reply:
@antonbakalets, I've fixed all the other issues, you've told me about, except this one. Could I, please, merge this brench in its current state for now without rewriting tests, because I'm limited in time and rewrite tests later as a new task? If everything else is correct, of course.

Reply:
You can do whatever you wish as soon as you have 2 approvals.

Reply:
Okay, got it, thank you.

Reply:
Okay, got it, thank you.